sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -30,258 +32,43 @@ namespace Azure.Messaging.EventHubs.Processor.Tests
     /// </remarks>
     ///
     [NonParallelizable]
+    [TestFixture]
     public class DiagnosticsTests
     {
-        /// <summary>The name of the diagnostics source being tested.</summary>
```

Comment:
The tests removed now belong to the diagnostics for `EventProcessor<T>`,  functionality and test coverage were not altered.